destination_config.go

```diff
@@ -50,7 +50,7 @@ type destinationConfigRequest struct {
 
 type DestinationConfigResponse struct {
 	Host                 string `json:"host"`
-	Port                 string `json:"port"`
+	Port                 string `json:"port"` // Request `int`, Response `string`
```

Comment:
Can you please explain the meaning of the comment here?

Reply:
To explain the request type != response type. See https://github.com/fivetran/go-fivetran/blob/main/destination_config.go#L30. 

We do something similar here:
- https://github.com/fivetran/go-fivetran/blob/main/destination_config.go#L58
- https://github.com/fivetran/go-fivetran/blob/main/destination_config.go#L35

I'll add more verbose comments in the next release. Thanks @shirolimit !